swift-blackjack-viewsTests/DeckSpec.swift

```diff
@@ -0,0 +1,48 @@
+//  DeckSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class DeckSpec: QuickSpec {
+    override func spec() {
+        describe("Deck") {
+            
+            var deck : Deck = Deck()
+            
+            beforeEach({ () -> () in
+                deck = Deck()
+            })
+            
+            describe("initializer") {
+                it("should set up the deck with 52 undealt cards") {
+                    expect(deck.description).to(contain("Cards Remaining: 52"))
+                    expect(deck.description).to(contain("Cards Dealt: 0"))
+                }
+            }
+            
+            describe("drawCard") {
+                it("should return the King of Clubs from a fresh deck") {
```

Comment:
Hm.. so it relies on the order of the valid cards/suits arrays? That feels icky, unless we make it clear in the instructions that it's important. I'd prefer not doing it... and the shuffle check should essentially just like... take the top card (or 3), shuffle, and hope that it's not the same top (3) card(s).